packages/itwin/one-click-lca-widget/src/components/context/OCLCAJobsClientContext.ts

```diff
@@ -0,0 +1,28 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { CARBON_CALCULATION_BASE_PATH, OCLCAJobsClient } from "@itwin/insights-client";
```

Comment:
`OCLCAJobsClient` -> `IOCLCAJobsClient`

Reply:
You will actually need both here. Return type of `createDefaultOCLCAJobsClient` should be the interface, but to create the value you will need the concrete class.

Reply:
Looks like `IOCLCAJobsClient` has not been exported from package.
https://github.com/iTwin/insights-client/blob/main/src/carbon-calculation/index.ts
I will try to raise a PR for insights-client to export `IOCLCAJobsClient`.